BuddySave/Core/GameSaveSyncManager.cs

```diff
@@ -1,51 +1,44 @@
 using BuddySave.Core.Models;
 using BuddySave.FileManagement;
-using NLog;
+using Microsoft.Extensions.Logging;
 
 namespace BuddySave.Core;
 
-public class GameSaveSyncManager : IGameSaveSyncManager
+public class GameSaveSyncManager(
+    ILogger<GameSaveSyncManager> logger, 
+    ISaveCopier saveCopier, 
+    IBackupManager backupManager)
+    : IGameSaveSyncManager
 {
-    private readonly ILogger _logger;
-    private readonly ISaveCopier _saveCopier;
-    private readonly IBackupManager _backupManager;
-
-    public GameSaveSyncManager(ILogger logger, ISaveCopier saveCopier, IBackupManager backupManager)
-    {
-        _logger = logger;
-        _saveCopier = saveCopier;
-        _backupManager = backupManager;
-    }
-
     public void UploadSave(GameSave gameSave)
     {
-        _saveCopier.ValidateSource(gameSave.SaveName, gameSave.LocalPath);
-        _backupManager.BackupFiles(gameSave.CloudPath, gameSave.GameName, gameSave.SaveName, SaveType.Cloud);
+        saveCopier.ValidateSource(gameSave.SaveName, gameSave.LocalPath);
```

Comment:
You've also turen to the dark side with naming fields without underscore? Fair dues.. 😄

Reply:
It's because of now used primary constructors in classes. Rider showed a warning for the naming convention on it so I've updated it accordingly :)

Reply:
Yeah, there's no longer a naming convention regarding private fields.. :( so I approve the change. :)